connectors/github/element-templates/github-connector.json

```diff
@@ -2279,6 +2561,22 @@
       },
       "optional": true
     },
+    {
+      "group": "input",
+      "type": "Hidden",
+      "value": "=context merge({\"message\": commitMessage, \"content\": b64content}, if sha != null then {\"sha\": sha} else {}, if branch != null then {\"branch\": branch} else {}, if committerName != null and committerEmail != null then {\"committer\": {\"name\": committerName, \"email\": committerEmail, \"date\": committerDate}} else {}, if authorName != null and authorEmail != null then {\"author\": {\"name\": authorName, \"email\": authorEmail, \"date\": authorDate }} else {})",
```

Comment:
If the branch is set to null instead of being absent, github will return a 422, so I had to get a bit creative here.
Idk if we prefer this pattern over `context from (get entries(context)[value != null])
I couldn't find either pattern in the codebase